src/cli/serve.rs

```diff
@@ -64,6 +64,7 @@ pub struct Args {
 
     /// Don't update all headers in one CKB transaction,
     /// to avoid size limit or cycles limit.
+    /// `0` means no limit.
```

Comment:
Personally, I feel `0 means no limit` is a bit special. Without special treatment, it should mean forbidden.

Another way to change（for reference only）

```rust
storage.generate_spv_client_and_spv_update(spv_tip_height, None)? ;
```

```rust
  if let Some(limit) = limit {
      tip_height = std::cmp::min(tip_height, prev_height.saturating_add(limit));
  }
```